nnpdf_data/nnpdf_data/new_commondata/ATLAS_TTBAR_13TEV_LJ_DIF/filter.py

```diff
@@ -1,7 +1,59 @@
 import yaml
-from validphys.commondata_utils import symmetrize_errors as se
-from validphys.commondata_utils import percentage_to_absolute as pta
-from validphys.commondata_utils import covmat_to_artunc as cta
```

Comment:
Such a function for sure should be part of the utils and for the purpose of this PR it should be still called from `validphys.commondata_utils`.